sphinx_selective_exclude/eager_only.py

```diff
@@ -31,9 +32,14 @@ def run(self, *args):
         env = self.state.document.settings.env
         env.app.builder.tags.add('TRUE')
         #print(repr(self.arguments[0]))
-        if not env.app.builder.tags.eval_condition(self.arguments[0]):
-            return []
-
+        try:
+            if not env.app.builder.tags.eval_condition(self.arguments[0]):
+                return []
+        except Exception as err:
+            logger = sphinx.util.logging.getLogger(__name__)
+            logger.warning("exception while evaluating 'only' directive expression: %s", err,
+                            location=(self.env.docname, self.lineno))
+            raise
```

Comment:
```suggestion
            logger.critical("Exception while evaluating 'only' directive expression: %r", err,
                            location=(self.env.docname, self.lineno))
            raise err
```
Addresses https://github.com/pfalcon/sphinx_selective_exclude/pull/7#discussion_r338610534